mysql-test/suite/binlog/t/binlog_mysqlbinlog_warn_stop_datetime.inc

```diff
@@ -0,0 +1,106 @@
+#
+# Helper file that ensures mysqlbinlog --stop-datetime behavior for local
+# files or a remote server
+#
+# Parameters:
+#   use_local_files (bool): A boolean that changes which source to use for
+#   mysqlbinlog. When true, uses local files; when false, reads remotely.
```

Comment:
A nit: the formatting between `binlog_mysqlbinlog_warn_stop_datetime.inc` and `binlog_mysqlbinlog_warn_stop_position.inc` are not consistent.

In fact, I wonder if we could combine the two includes into one file?
(`binlog_mysqlbinlog_warn_stop_datetime.test` and `binlog_mysqlbinlog_warn_stop_position.test` has different generators that can’t easily combine.)

Reply:
We could combine them, though I think for the scope of this patch it is fine. Let's plan to do that in the follow up patch which will fix `--stop-position` warnings to work with `--read-from-remote-server`.